tutoraspects/templates/aspects/apps/aspects/migrations/alembic/versions/0015_event_sink_user_profile.py

```diff
@@ -0,0 +1,47 @@
+"""
+create user_profile sink table
+"""
+from alembic import op
+import sqlalchemy as sa
+
+revision = "00015"
+down_revision = "0014"
+branch_labels = None
+depends_on = None
+
+def upgrade():
+    op.execute(
+        """
+        CREATE TABLE IF NOT EXISTS {{ ASPECTS_EVENT_SINK_DATABASE }}.{{ ASPECTS_EVENT_SINK_USER_PROFILE_TABLE }}
+        (
+            id Int32 NOT NULL,
+            user_id Int32 NOT NULL,
+            name String NOT NULL,
+            meta String NOT NULL,
+            courseware String NOT NULL,
+            language String NOT NULL,
+            location String NOT NULL,
+            year_of_birth String NOT NULL,
+            gender String NOT NULL,
+            level_of_education String NOT NULL,
+            mailing_address String NOT NULL,
+            city String NOT NULL,
+            country String NOT NULL,
+            state String NOT NULL,
+            goals String NOT NULL,
+            bio String NOT NULL,
+            profile_image_uploaded_at String NOT NULL,
+            phone_number String NOT NULL,
+            dump_id UUID NOT NULL,
+            time_last_dumped String NOT NULL
+        ) engine = MergeTree
+        PRIMARY KEY (id, time_last_dumped)
+        ORDER BY (id, time_last_dumped);
```

Comment:
@SoryRawyer can you take a look at this table?

Reply:
@Ian2012 could you point me to the source table for this? I'm not familiar with the process that creates this data.

Reply:
Sure, there is the PR that added that functionality for the `openedx-event-sink-clickhouse` plugin https://github.com/openedx/openedx-event-sink-clickhouse/pull/28